src/server/db/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "development": {
+    "username": "user",
+    "password": "password",
+    "database": "leket",
+    "host": "localhost",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "test": {
+    "username": "root",
+    "password": null,
+    "database": "leket",
+    "host": "192.168.99.100",
+    "port": 5433,
+    "dialect": "postgres"
+  },
+  "production": {
+    "use_env_variable": "DATABASE_URL"
```

Comment:
IMO it's easier to have this file as `.js` (instead of `.json`) and do:

```
module.exports = {
  "development": {...},
  "test": {...},
  "production": {
    "host": process.env.HOST,
    ...
}
```

WDYT?